### PR TITLE
Fix: App crash on joining active hike due to google maps 

### LINKED ISSUE #205: Android app crashes with SIG 9
**Describe the bug**
App crash due to IncompatibleClassChangeError when using FusedLocationProviderClient.
Screenshot 2023-04-03 at 12 54 18 PM

**To Reproduce**
The app crashes when trying to join an active hike.

**Demonstration**
https://drive.google.com/file/d/1-LT_qQTpnhri7WI0DRp4xG1wz8CtC3n8/view

**Environment**
- OS (Android): Android 9
- Flutter version: Flutter 3.7.8

**Are you working on this issue? (Yes)**

### PR DESCRIPTION
**Pull request by tanisha083**
Fixes #205 

Describe the changes you have made in this PR -
The error log suggests an IncompatibleClassChangeError in the com.example.beacon process, with a message indicating that the code found an interface(com.google.android.gms.location.FusedLocationProviderClient) where a class was expected. This error can occur when different versions of the same library are used in the project, or when there is a mismatch between the libraries used by different modules or libraries. To fix this error below-mentioned change were made:

- The resolutionStrategy block is used to force the version of the com.google.android.gms:play-services-location library to 21.0.1 for all dependencies in the project to resolve any conflicting dependencies.

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

### CODE REVIEW

File: android/gradle.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
+android.enableDexingArtifactTransform=false
```
Comment (Techno-Disaster) [inline android/gradle.properties:4]: why do we need this? https://github.com/Lyokone/flutterlocation/issues/802

Comment (tanisha083) [reply android/gradle.properties:4]: While I was debugging I saw an issue related to flutter google maps (https://github.com/flutter/flutter/issues/72185) so I added this as an extra check.

Comment (Techno-Disaster) [reply android/gradle.properties:4]: This looks very old, can you remove the line test and confirm if it works without it? Thanks :3

Comment (tanisha083) [reply android/gradle.properties]: Removed the line, works well without it as well. Thanks

**Pull request merged**